client/src/components/ReviewList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import StyledReview from './Review.jsx';
+import styled from 'styled-components';
+
+const FlexRow = styled.div`
+display: flex;
+flex-direction: row;
+`;
+
+const ReviewList = ( {reviews} ) => {
```

Comment:
Change to dynamically render reviews, not hard coded

Reply:
https://github.com/keybox130/Reviews-Service/pull/4/commits/591f35faf10efd8be164ad4091f3b20ff885cb21